README.md

```diff
@@ -32,64 +28,11 @@ development environment for each component.
 
 ## Contributing
 
-We'd love to accept your patches and contributions to this project. For more
-information, including details on the required Contributor License Agreement
-(CLA), code reviews, and environment setup, see
-[Contributing to Ground Platform](CONTRIBUTING.md).
-
-## Getting started
```

Comment:
Do we have some of these instructions elsewhere still? Seems important for testing locally.

Reply:
There are instructions in READMEs under each subdirectory and in #1028 which I plan merge these into this one.